examples/nest/Potjans_2014/network_params.sli

```diff
@@ -101,8 +101,40 @@ def
 /neuron_model /iaf_psc_exp def  % neuron model. For PSP-to-PSC conversion to
                                 % be correct, synapses should be current-based
                                 % with an exponential time course
-/Vm0_mean -58.0 def             % mean of initial membrane potential (mV)
-/Vm0_std 10.0 def               % std of initial membrane potential (mV)
+
+% Initial conditions for the membrane potential, options are:
+% (original): uniform mean and std for all populations
+% (optimized): population-specific mean and std, allowing a reduction of the
+% initial activity burst in the network
+/Vm0_type (optimized) def        % choose either (original) or (optimized)
```

Comment:
I think that the default value should be set to `original`, as users might not be aware that the example has changed.